install/installing-kata-static-binaries.md

```diff
@@ -0,0 +1,47 @@
+# Install Kata Containers static binaries
+
+* [Overview](#overview)
+* [Installation](#installation)
+
+# Overview
+
+Kata Containers is available as an archive file containing a set of static
+binaries. This alternative installation is useful for cloud provisioning
+systems which automatically pull the latest static binaries release.
+
+We recommend normal users install using the
+[automatic installation method](https://github.com/kata-containers/documentation/tree/master/install)
+unless they understand the full implications of installing using the static binaries.
+
+> **WARNING:**
+>
+> This installation method is **not** recommended for normal users since the
+> installation does not use the distribution package manager. This means new
+> versions of Kata Containers will **not** be automatically installed.
```

Comment:
Line 20 suggested rewrite (for an active voice): 
versions of Kata Containers will **not** automatically install.

Reply:
Thanks @klynnrif - branch updated.